docs/src/reference/5_intersection_problems.md

```diff
@@ -0,0 +1,29 @@
+# Intersection problems
+
+Problems over intersections of convex sets, i.e. 
+```math
+\min_{x \in \bigcap_{i=1}^n P_i} f(x),
```

Comment:
the paper referenced below focuses on feasibility problems (finding any point in the intersection).
We should probably refer to the Beck, A., Pauwels, E., and Sabach, S. paper for the problem of minimization over the intersection.